src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -1,73 +1,10 @@
-open Core_kernel
 open Async_kernel
-open Protocols.Coda_pow
 open Coda_base
-open Signature_lib
+open Core_kernel
+open Protocols.Coda_pow
 
 module type Inputs_intf = sig
-  module Staged_ledger_aux_hash : Staged_ledger_aux_hash_intf
-
-  module Ledger_proof_statement :
-    Ledger_proof_statement_intf with type ledger_hash := Frozen_ledger_hash.t
-
-  module Ledger_proof : sig
-    include
-      Ledger_proof_intf
-      with type statement := Ledger_proof_statement.t
-       and type ledger_hash := Frozen_ledger_hash.t
-       and type proof := Proof.t
-       and type sok_digest := Sok_message.Digest.t
-
-    include Binable.S with type t := t
-
-    include Sexpable.S with type t := t
-  end
-
-  module Transaction_snark_work :
-    Transaction_snark_work_intf
-    with type proof := Ledger_proof.t
-     and type statement := Ledger_proof_statement.t
-     and type public_key := Public_key.Compressed.t
-
-  module Staged_ledger_diff :
-    Staged_ledger_diff_intf
-    with type user_command := User_command.t
-     and type user_command_with_valid_signature :=
-                User_command.With_valid_signature.t
-     and type staged_ledger_hash := Staged_ledger_hash.t
-     and type public_key := Public_key.Compressed.t
-     and type completed_work := Transaction_snark_work.t
-     and type completed_work_checked := Transaction_snark_work.Checked.t
-     and type fee_transfer_single := Fee_transfer.single
-
-  module External_transition :
-    External_transition.S
-    with module Protocol_state = Consensus.Protocol_state
-     and module Staged_ledger_diff := Staged_ledger_diff
-
-  module Staged_ledger :
-    Staged_ledger_intf
-    with type diff := Staged_ledger_diff.t
-     and type valid_diff :=
-                Staged_ledger_diff.With_valid_signatures_and_proofs.t
-     and type staged_ledger_hash := Staged_ledger_hash.t
-     and type staged_ledger_aux_hash := Staged_ledger_aux_hash.t
-     and type ledger_hash := Ledger_hash.t
-     and type frozen_ledger_hash := Frozen_ledger_hash.t
-     and type public_key := Public_key.Compressed.t
-     and type ledger := Ledger.t
-     and type ledger_proof := Ledger_proof.t
-     and type user_command_with_valid_signature :=
-                User_command.With_valid_signature.t
-     and type statement := Transaction_snark_work.Statement.t
-     and type completed_work_checked := Transaction_snark_work.Checked.t
-     and type sparse_ledger := Sparse_ledger.t
-     and type ledger_proof_statement := Ledger_proof_statement.t
-     and type ledger_proof_statement_set := Ledger_proof_statement.Set.t
-     and type transaction := Transaction.t
-     and type user_command := User_command.t
-
-  val max_length : int
+  include Transition_frontier0.Inputs_intf
```

Comment:
You can basically inline the usages of `Inputs_intf` with `Transition_frontier0.Inputs_intf`

Reply:
I wanted to re-expose the interface here because many other files use Transition_frontier.Inputs_intf, but I can probably just alias it because we don't need to add anything for now